CDP4Composition/Modularity/PluginUtilities.cs

```diff
@@ -26,27 +26,40 @@
 
 namespace CDP4Composition.Modularity
 {
+    using System;
     using System.Collections.Generic;
     using System.Diagnostics.CodeAnalysis;
     using System.IO;
+    using System.IO.Compression;
+    using System.Linq;
+    using System.Reflection;
 
     using CDP4Composition.Utilities;
 
     using Microsoft.Practices.ServiceLocation;
 
     using Newtonsoft.Json;
 
+    using NLog;
+
     /// <summary>
     /// Utility class that help with plugin management
     /// </summary>
     [ExcludeFromCodeCoverage]
     public static class PluginUtilities
```

Comment:
can you let me know why this class is excluded from testing?

Reply:
Alexander had to add the pre-processor condition in the `GetPluginManifests()` method which can not be covered in the same test configuration.